ios/Video/RCTVideo.swift

```diff
@@ -1177,6 +1183,14 @@ class RCTVideo: UIView, RCTVideoPlayerViewControllerDelegate, RCTPlayerObserverH
             _pendingSeek = false
         }
 
+        if _startPosition > 0 {
+            setSeek([
+                "time": NSNumber(value: _startPosition),
+                "tolerance": NSNumber(value: 100)
+            ])
+            _startPosition = -1
```

Comment:
is it really need to force _startPosition to -1 ?

Reply:
I was assuming that the playback would work fine, then an error would occur, and then it would work again.

When the source is unchanged and the lifecycle is as follows, 
```
(1) readyToPlay → (2) failed → (3) readyToPlay
```
I forced it to `-1` because I would expect `startPosition` to not work in state (3).

But I'm not very experienced with iOS, and I don't know much about it. If my assumptions are wrong, feel free to let me know and I'll correct above code.